spec/features/residents_see_nice_errors_spec.rb

```diff
@@ -29,7 +29,7 @@
 
   scenario 'when a page was not found' do
     visit '/422'
-    expect(page).to have_content "We're sorry, but the change you wanted was rejected"
+    expect(page).to have_content "We're really sorry"
```

Comment:
While this checks we are looking at the right page it seems to me that the meaningful message on the page is in fact:

> Something seems to have gone wrong while reporting your repair.

It would be more helpful in this case if our tests told us if this part was accidentally removed, over the header.